src/main/webapp/app/entities/result.model.ts

```diff
@@ -11,6 +11,11 @@ export class Result implements BaseEntity {
     public completionDate?: dayjs.Dayjs;
     public successful?: boolean;
     public hasFeedback?: boolean;
+
+    /*
```

Comment:
```suggestion
    /**
```
This way it should be displayed when hovering over the variable